src/engine/bank02.asm

```diff
@@ -1,5 +1,58 @@
-Func_8000: ; 8000 (2:4000)
-	INCROM $8000, $8211
+DuelCheckInterface: ; 8000 (2:4000)
```

Comment:
The way I would do this is: rename this one to``_DuelCheckInterface`` and then ``CallDuelCheckInterface`` to ``DuelCheckInterface``

Reply:
Got it, I fixed it in the newest commit.